pkg/metaserver/agent/metric/provisioner/malachite/provisioner.go

```diff
@@ -169,6 +169,11 @@ func (m *MalachiteMetricsProvisioner) updateCgroupData() error {
 	cgroupPaths := []string{m.baseConf.ReclaimRelativeRootCgroupPath, common.CgroupFsRootPathBurstable, common.CgroupFsRootPathBestEffort}
 	errList := make([]error, 0)
 	for _, path := range m.baseConf.GeneralRelativeCgroupPaths {
```

Comment:
The OptionalRelativeCgroupPaths is a general configuration of malichite, and other components may depend on this configuration in the future. It is not reasonable to ignore non-existent cgroup paths by default. Please add an OptionalRelativeCgroupPaths for this case.